internal/iam/command_rbac_role_binding_list.go

```diff
@@ -491,6 +510,10 @@ func (c *roleBindingCommand) ccloudListRolePrincipals(cmd *cobra.Command, listRo
 			row.Name = name
 			list.Add(row)
 		}
+		if name, ok := groupMappingToNameMap[principal]; ok {
```

Comment:
Unfortunate that we need to keep adding to this `for` loop every time we introduce a new resource.